CHANGELOG.md

```diff
@@ -68,6 +68,8 @@
   - The 'json' option is no longer supported as the 'csv' option provides better performance (PR#2372 by elsif2).
 - `intelmq.bots.collectors.alienvault_otx.collector` (PR#2449 by qux-bbb):
   - Fix modified_pulses_only is always False.
+- `intelmq.bots.collectors.shodan.collector_stream` (PR#2492 by Mikk Margus Möll):
+  - Add `alert` parameter to Shodan stream collector to allow fetching streams by configured alert ID
```

Comment:
It's in the wrong section, the section for the current release was missing. I just merged https://github.com/certtools/intelmq/pull/2490 to add the template. Could you please rebase it on current develop and move the entry?